source/guides/kernel/kernel-boot-msg.rst

```diff
@@ -0,0 +1,62 @@
+.. _kernel-boot-msg:
+
+Capture Kernel Boot Messages in the Journal
+###########################################
+
+On |CL|, by default, kernel boot messages are not captured in the journal 
+and reported as "Missed" messages.  This is done in order to provide fast boot 
```

Comment:
`This` pronoun gets lost in the shuffle. Consider something like: 

By default |CL| does not capture kernel boot messages in the journal logs, where they're reported as "Missed" messages.  This design decision for the  [boot-loader?] was made to provide a fast boot time. On the other hand, if you wish to see the message, follow this guide.

Something along those lines. Does that make sense?

Reply:
made some changes per your suggestion.

Reply:
my bad.  i forgot to "add."  

for some reason, the automated validation process is taking much longer than usual.